src/api/app/models/concerns/project_distribution.rb

```diff
@@ -0,0 +1,21 @@
+module ProjectDistribution
+  extend ActiveSupport::Concern
+
+  # Check if the project has a path_element matching project and repository
+  def has_distribution(project_name, repository)
+    has_local_distribution(project_name, repository) || has_remote_distribution(project_name, repository)
+  end
+
+  def has_remote_distribution(project_name, repository)
```

Comment:
I know the original code doesn't document those two methods. It would however be good to have it documented, either as a code (spec) or inline comment like the method `has_distribution`

Reply:
It's documented on line 4 no?